nullaway/src/main/java/com/uber/nullaway/NullAway.java

```diff
@@ -1096,15 +1131,121 @@ private Description createErrorDescription(
       Tree errorLocTree, String message, @Nullable Tree suggestTree) {
     Description.Builder builder = buildDescription(errorLocTree).setMessage(message);
     if (config.suggestSuppressions() && suggestTree != null) {
-      builder = addSuppressWarningsFix(suggestTree, builder);
+      if (message.contains(MessageTypes.DEREFERENCE_NULLABLE.toString())
```

Comment:
Why don't we pass in the `MessageTypes` value separately, instead of concatenating it to the error message and then parsing it back out again?